File: Unity/Scripts_Unity/Scripts/Ejercicio_2.cs
using System.Numerics;
using UnityEngine;
using Vector3 = UnityEngine.Vector3;

public class Ejercicio_2 : MonoBehaviour
{
    [SerializeField] private float speed = 5;

    [SerializeField] private float topLimit;

    [SerializeField] private float downLimit;

    private Vector3 direction = Vector3.up;

    // Update is called once per frame
    void Update()
    {
        Vector3 currectCoords = transform.position;
        if (currectCoords.y > topLimit || currectCoords.y < downLimit)
        {
            direction.y *= -1;
        }

        Vector3 velocity = direction * speed;
        Vector3 displacement = velocity * Time.deltaTime;
        transform.position += displacement;
    }
}

File: Programacion/1-Trimestre/Tema3.Java/Ejercicios_Basicos_Bucles/Ejercicio_2.java
public class Ejercicio_2 {
    public static void main(String[] args) {

        int i = -10;
        System.out.println("While");
        while (i <= -1) {
            System.out.println(i);
            i++;
        }

        i = -10;

        System.out.println("Do While");
        do {
            System.out.println(i);
            i++;
        } while (i <= -1);

        System.out.println("For");
        for (i = -10; i <= -1; i++) {
            System.out.println(i);
        }

    }
}
